docs/source/models/gbd2017/causes/ischemic_heart_disease/index.rst

```diff
@@ -53,6 +81,22 @@ Restrictions
 
    Describe more assumptions and limitations of the model.
 
+Cause Model Diagram
+--------------------
+.. image:: ischemic_heart_disease_transitions.svg
+  :width: 150
```

Comment:
This is pretty small and hard to see. If you just leave out the width, I think the image will automatically scale to a reasonable size.

Reply:
Thank you for the input! Will do that before merging with master.